src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -98,14 +101,21 @@ MulticopterAttitudeControl::parameters_updated()
 float
 MulticopterAttitudeControl::throttle_curve(float throttle_stick_input)
 {
-	// throttle_stick_input is in range [0, 1]
+	float thrust = 0.f;
+
 	switch (_param_mpc_thr_curve.get()) {
 	case 1: // no rescaling to hover throttle
-		return math::interpolate(throttle_stick_input, 0.f, 1.f, _param_mpc_manthr_min.get(), _param_mpc_thr_max.get());
+		thrust = math::interpolate(throttle_stick_input, -1.f, 1.f,
+					   _manual_throttle_minimum.getState(), _param_mpc_thr_max.get());
+		break;
 
 	default: // 0 or other: rescale to hover throttle at 0.5 stick
```

Comment:
You're right, I didn't even spot that since I left it unchanged. How about:
```suggestion
	default: // 0 or other: rescale such that a centered throttle stick corresponds to hover throttle
```
?